trip-ninja-docs/src/common/ConnectWithSales.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./Common.css";
+
+
+export default function ConnectWithSales() {
+  return(
+    <div className="row connect-banner">
+      <div className="col-8">
+        <h3>Not set up with Trip Ninja?</h3>
+        <p className="body-text mb-0">
+          Don’t miss out on the unbeatable results from our API, talk to our sales 
+          team to kickstart your Trip Ninja experience.
+        </p>
+      </div>
+      <div className="col-4 my-auto text-end">
+        <a 
+          href="mailto:support@tripninja.io?subject=Connect with Sales - Developer Questions" 
```

Comment:
more of a question than a comment but what's the desired behaviour of opening a page when clicking on this link? In my Navbar branch i have it opening to a new tab by setting `target="_blank"` in the <a> tag. Not sure which is best, new tab or same tab?

Reply:
Yeah I think opening in new tab is best!